### ISSUE TITLE
Сообщения собеседника не анализируются

### ISSUE THREAD

**Issue by annkomkova** [vlajnaya-mol/message-analyser#15]
Все сообщения собеседника игнорируются, дальнейший анализ и категоризация — тоже.

<img width="668" alt="Снимок экрана 2022-09-04 в 17 07 10" src="https://user-images.githubusercontent.com/71514824/188317715-6045ab5c-37e6-467b-978d-8956d5241526.png">

**Comment by PoSayDone** [comment]
Да, абсолютно такая же проблема, попробовал и на большом и на маленьком диалоге, везде не работает.

**Comment by nagayev** [comment]
@annkomkova  @PoSayDone Пофиксил.
Можете проверить мою версию или подождать пока это изменение включат в мастер.
